### PR TITLE
[JA] Update tutorials/quickstart/beginner.ipynb

### PR DESCRIPTION
**Pull request by AseiSugiyama**
The original document is https://github.com/tensorflow/docs/blob/master/site/en/tutorials/quickstart/beginner.ipynb

This PR updates beginner.ipynb since it is outdated.

### CODE REVIEW

**Comment by AseiSugiyama** [comment]
@lamberta I updated about half of the translated notebooks, so I would like to wait for a review before merging into the master.

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 1]
```diff
@@ -180,11 +180,7 @@
         "  tf.keras.layers.Dense(128, activation='relu'),\n",
         "  tf.keras.layers.Dropout(0.2),\n",
         "  tf.keras.layers.Dense(10, activation='softmax')\n",
```
Comment (kiszk) [inline site/ja/tutorials/quickstart/beginner.ipynb:182]: [Suggestion]
```suggestion
        "  tf.keras.layers.Dense(10)\n",
```

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 2]
```diff
@@ -207,19 +203,200 @@
       },
       "outputs": [],
       "source": [
-        "model.fit(x_train, y_train, epochs=5)\n",
+        "predictions = model(x_train[:1]).numpy()\n",
+        "predictionspredictions = model(x_train[:1]).numpy()\n",
```
Comment (kiszk) [inline site/ja/tutorials/quickstart/beginner.ipynb:207]: [suggestion] This line does not exist in https://raw.githubusercontent.com/tensorflow/docs/master/site/en/tutorials/quickstart/beginner.ipynb

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 3]
```diff
@@ -207,19 +203,200 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "`tf.nn.softmax` 関数はこれらのロジットをそれぞれのクラスにである\"確率\"に変換します。"
```
Comment (kiszk) [inline site/ja/tutorials/quickstart/beginner.ipynb:218]:
```suggestion
        "`tf.nn.softmax` 関数はこれらのロジットをそれぞれのクラスにである\"確率\"に変換します。"
[weak suggestion
```
```suggestion
        "`tf.nn.softmax` 関数はこれらのロジットをそれぞれのクラスに関する\"確率\"に変換します。"
```
or others.

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 4]
```diff
@@ -207,19 +203,200 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "dhVMTSPXh5cs"
+      },
+      "source": [
+        "`losses.SparseCategoricalCrossentropy` 損失関数はロジットからなるベクトルと `True` を入力にとり、それぞれの標本についてクラスごとに損失のスカラーを返します。"
```
Comment (kiszk) [inline site/ja/tutorials/quickstart/beginner.ipynb:251]: [weak suggestion]
```suggestion
        "`losses.SparseCategoricalCrossentropy` 損失関数は、ロジットと `True` の インデックスに関するベクトルを入力にとり、それぞれの標本についてクラスごとに損失のスカラーを返します。"
```

Based on `scce(y_true, y_pred).numpy()` at https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 1 above)
Comment (AseiSugiyama) [reply site/ja/tutorials/quickstart/beginner.ipynb:182]: Thank you for your review! I'll fix this issue by applying the `nb_code_sync.py` tool.

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 2 above)
Comment (AseiSugiyama) [reply site/ja/tutorials/quickstart/beginner.ipynb:207]: Same as above, I'll fix this issue by applying the `nb_code_sync.py` tool.

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 3 above)
Comment (AseiSugiyama) [reply site/ja/tutorials/quickstart/beginner.ipynb:218]: Thank you for your suggestion. I'd like to change this line to "`tf.nn.softmax` 関数はクラスごとにこれらのロジットを "確率" に変換します。"

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 4 above)
Comment (AseiSugiyama) [reply site/ja/tutorials/quickstart/beginner.ipynb:251]: Thanks, I'd like to use your suggestion.

**Comment by AseiSugiyama** [comment]
@kiszk I made some changes after your review, could you check it again?

**Comment by kiszk** [comment]
Thank you, LGTM
